htdocs/admin/modulehelp.php

```diff
@@ -129,7 +129,7 @@
 								if (preg_match('/deprecated/', $objMod->version) && (empty($conf->global->$const_name) && ($conf->global->MAIN_FEATURES_LEVEL >= 0))) $modulequalified=0;
 
 		    					// We discard modules according to property disabled
-		    					if (! empty($objMod->hidden)) $modulequalified=0;
+		    					//if (! empty($objMod->hidden)) $modulequalified=0;
```

Comment:
Why removing this line ?

Reply:
the modulehelp display context is not available in the module section. The hidden module does not offer links to moduelhelp.
However by removing this line, it gives the possibility to display all module from its id without further restriction. the restrictions and filter belonging to the modulehelp page.
Finally, in the construction of our module (oscss-shop) we use plugins (plugin) module based on secondary descriptor contained in a given module.
So that these do not disturb the main display of the module page, we declared them hidden, and they are only accessible through the main module.
deleting this line allows you to use moduelhelp to display the details of these descriptors.